paddle/framework/scope.h

```diff
@@ -65,6 +65,8 @@ class Scope {
   /// Drop all kids scopes belonged to this scope.
   void DropKids();
 
+  void EraseVars(std::vector<std::string>& var_names);
```

Comment:
Since our every variable will be released after we destruct the scope, In my humble view, this interface should never be used.